jzy3d-jGL/src/main/java/jgl/wt/swt/JGLCanvas.java

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * (c) 2021 Läubisoft GmbH, Christoph Läubrich
+ * 
+ * This library is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU Lesser General Public License as published by the Free Software Foundation; either version
+ * 2.1 of the License, or any later version. the GNU Lesser General Public License should be
+ * included with this distribution in the file LICENSE.
+ * 
+ * This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ */
+package jgl.wt.swt;
+
+import org.eclipse.swt.events.ControlEvent;
+import org.eclipse.swt.events.ControlListener;
+import org.eclipse.swt.graphics.GC;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.graphics.Point;
+import org.eclipse.swt.opengl.GLData;
+import org.eclipse.swt.widgets.Canvas;
+import org.eclipse.swt.widgets.Composite;
+
+public class JGLCanvas extends Canvas {
```

Comment:
Why not calling this jgl.wt.swt.GLCanvas to be similar to the jgl.wt.awt.GLCanvas ?

Reply:
There is already a swt class class [GLCanvas](https://javadoc.scijava.org/Eclipse/org/eclipse/swt/opengl/GLCanvas.html) and thus I'd like to avoid confusion with that class